README.md

```diff
@@ -1,21 +1,58 @@
-# react-2dplot
+# react-plot
 
 [![NPM version][npm-image]][npm-url]
 [![npm download][download-image]][download-url]
 
 Render 2dplot partition in react.
 
-## [Storybook](https://zakodium.github.io/react-2dplot/)
+## [Storybook](https://zakodium.github.io/react-plot/)
+
+```tsx
+const Example = () => (
+  <Plot
+    width={500}
+    height={500}
+    colorScheme={d3.schemePaired}
+    margin={{ right: 200 }}
+  >
+    <PlotHeading
```

Comment:
I would rename to Heading and remove the property 'top'. If it is a heading it should be on the top.

Reply:
If I have several plots I could want to have this on the bottom, and I like the `ComponentSubcomponent` style for subcomponents, in order to make it easy to understand when you imported (Header could be your layout header, menu header, etc)